Makefile

```diff
@@ -11,6 +11,9 @@ IMAGE_NAME := fidesops
 IMAGE := $(REGISTRY)/$(IMAGE_NAME):$(IMAGE_TAG)
 IMAGE_LATEST := $(REGISTRY)/$(IMAGE_NAME):latest
 
+DOCKERFILE_ENVIRONMENTS := postgres mysql mongodb mssql
+EXTERNAL_ENVIRONMENTS := snowflake redshift
```

Comment:
`ENVIRONMENT` is a really overloaded term and I think you are inviting trouble by using this and `ENV` as your variables. I'd recommend making it more specific, like `make pytest-integration INTEGRATION_DATABASES=postgres,mysql` or similar

Reply:
And add a comment above these magic variables to describe their use

Reply:
You also don't appear to be using EXTERNAL_ENVIRONMENTS.